server/runtime/src/main/java/org/infinispan/server/configuration/security/JwtConfiguration.java

```diff
@@ -19,10 +20,10 @@ public class JwtConfiguration extends ConfigurationElement<JwtConfiguration> {
    static final AttributeDefinition<String> CLIENT_SSL_CONTEXT = AttributeDefinition.builder(Attribute.CLIENT_SSL_CONTEXT, null, String.class).build();
    static final AttributeDefinition<String> HOST_NAME_VERIFICATION_POLICY = AttributeDefinition.builder(Attribute.HOST_NAME_VERIFICATION_POLICY, null, String.class).build();
    static final AttributeDefinition<String[]> ISSUER = AttributeDefinition.builder(Attribute.ISSUER, null, String[].class).build();
-   static final AttributeDefinition<Long> JKU_TIMEOUT = AttributeDefinition.builder(Attribute.JKU_TIMEOUT, null, Long.class).build();
+   static final AttributeDefinition<TimeQuantity> JKU_TIMEOUT = AttributeDefinition.builder(Attribute.JKU_TIMEOUT, TimeQuantity.valueOf("2m")).parser(TimeQuantity.PARSER).build();
```

Comment:
The 2m was the default before?

Reply:
Yes, that's the underlying default in Elytron